301-jenkins-vmss-blue-green/azuredeploy.json

```diff
@@ -0,0 +1,615 @@
+{
+    "$schema": "https://schema.management.azure.com/schemas/2015-01-01/deploymentTemplate.json#",
+    "contentVersion": "1.0.0.0",
+    "parameters": {
+        "resourceNamePrefix": {
+            "defaultValue": "vmssbg",
+            "type": "string",
+            "metadata": {
+                "description": "Prefix for the resource names."
+            }
+        },
+        "adminUsername": {
+            "defaultValue": "azureuser",
+            "type": "string",
+            "metadata": {
+                "description": "User name for the Virtual Machine hosting the Jenkins and the Virtual Machines in VMSS."
+            }
+        },
+        "sshPublicKey": {
+            "type": "string",
+            "metadata": {
+                "description": "Configure all linux machines with the SSH public key string, for example 'ssh-rsa AAAAB...snip...UcyupgH azureuser@linuxvm'"
+            }
+        },
+        "virtualMachineSize": {
+            "type": "string",
+            "defaultValue": "Standard_DS2_v2",
+            "allowedValues": [
+                "Standard_A2_v2",
+                "Standard_A2",
+                "Standard_B2s",
+                "Standard_B2ms",
+                "Standard_D2s_v3",
+                "Standard_D2_v3",
+                "Standard_DS2_v2",
+                "Standard_D2_v2",
+                "Standard_DS2",
+                "Standard_D2"
+            ],
+            "metadata": {
+                "description": "The virutal machine size to use. We picked out the sizes with 2 vCPUs, but in real world projects you can choose other sizes as you desired."
```

Comment:
add a note about requiring premium disk support

Reply:
I think we don't need premium disk support in the template. I will check the template to confirm.

Reply:
Updated the template so that we don't need premium storage. The comment here was also updated as we didn't provide the allowed size list now.